### ISSUE TITLE
FIDO U2F not supported on Firefox

### ISSUE THREAD

**Issue by R-Adrian** [jullrich/ISC-Bugs#32]
tl;dr version: https://isc.sans.edu/u2fregister.html
does not accept Firefox as U2F-aware.

context:
a) i'm using Firefox Quantum - and FIDO U2F is supported natively on Quantum, if the prefs are enabled, there's no more need of the separate U2F addon. (current version is 58.0.2)

b) under about:config i enabled 'security.webauth.u2f' and 'security.webauth.webauthn_enable_usbtoken' - this will enable U2F USB keys usage in firefox. (i'm using Yubico keys)

FIDO U2F works fine with Firefox on https://demo.yubico.com/u2f

related meta bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1065729
the next version of Firefox will also have an interface for these preferences.
